.github/workflows/lint.yml

```diff
@@ -15,20 +15,19 @@ jobs:
         with:
           fetch-depth: 1
           submodules: recursive
-      # ------------------------------------------------------------------------------
-      # Setup Node.
-      # ------------------------------------------------------------------------------
-      - name: Check for .nvmrc file
-        id: check-nvmrc
-        run: echo "::set-output name=exists::$(test -f ${{ github.workspace }}/.nvmrc && echo 'true' || echo 'false')"
-
-      - uses: actions/setup-node@v3
-        if: steps.check-nvmrc.outputs.exists == 'true'
+      - name: Set node version
```

Comment:
@Camwyn @bordoni Am I correct that the changes to this particular file are *not* desired?

Reply:
Yeah, we don't want that to change